tests/test_circuit_graph.py

```diff
@@ -160,192 +120,92 @@ def test_dependence(self, queue, obs):
-        """Test that `nx.set_node_attributes` is called correctly when updating a node using
-        `update_node` and a given node and op."""
-        mock_command = MagicMock()
-        mock_nx = MagicMock()
-
-        monkeypatch.setattr("pennylane.circuit_graph.nx", mock_nx)
-        monkeypatch.setattr("pennylane.circuit_graph.Command", mock_command)
-
-        node = MagicMock()
-        op = MagicMock()
-
-        circuit.update_node(node, op)
-
-        mock_nx.set_node_attributes.assert_called_once_with(
-            circuit._graph,
-            {
-                node["idx"]: {
-                    **mock_command(
-                        name=op.name, op=op, return_type=op.return_type, idx=node["idx"]
-                    )._asdict()
-                }
-            },
-        )
+    def test_update_node(self, ops):
+        """Changing nodes in the graph."""
+
+        circuit = CircuitGraph(ops, {})
+        new = qml.RX(0.1, wires=0, do_queue=False)
+        circuit.update_node(ops[0], new)
+        assert circuit.operations[0] is new
 
     def test_observables(self, circuit, obs):
         """Test that the `observables` property returns the list of observables in the circuit."""
         assert circuit.observables == obs
 
-    def test_observable_nodes_and_operation_nodes(self, circuit, monkeypatch):
-        """Test that the `observable_nodes` and `operation_nodes` methods return the correct
-        values."""
-        mock_graph = MagicMock()
-        mock_graph.nodes.values.return_value = [
-            {"return_type": None, "idx": 0},
-            {"return_type": MagicMock(), "idx": 1},
-        ]
-        monkeypatch.setattr(circuit, "_graph", mock_graph)
-        assert circuit.observable_nodes == [mock_graph.nodes.values()[1]]
-        assert circuit.operation_nodes == [mock_graph.nodes.values()[0]]
-
     def test_operations(self, circuit, queue):
         """Test that the `operations` property returns the list of operations in the circuit."""
         assert circuit.operations == queue
 
-    def test_get_names(self, circuit):
-        result = circuit.get_names(range(9))
-        expected = ["RX", "RY", "RZ", "CNOT", "Hadamard", "CNOT", "PauliX", "PauliX", "Hermitian"]
-        assert result == expected
-
     def test_op_indices(self, circuit):
         """Test that for the given circuit, this method will fetch the correct operation indices for
         a given wire"""
         op_indices_for_wire_0 = [0, 3, 5, 7]
         op_indices_for_wire_1 = [1, 3, 6, 8]
         op_indices_for_wire_2 = [2, 4, 5, 8]
 
-        assert circuit.get_wire_indices(0) == op_indices_for_wire_0
-        assert circuit.get_wire_indices(1) == op_indices_for_wire_1
-        assert circuit.get_wire_indices(2) == op_indices_for_wire_2
+        assert circuit.wire_indices(0) == op_indices_for_wire_0
+        assert circuit.wire_indices(1) == op_indices_for_wire_1
+        assert circuit.wire_indices(2) == op_indices_for_wire_2
```

Comment:
lol this is the third time this method has been renamed in 3 PRs, and this is back to the original name now.

Reply:
And maybe the method isn't needed at all :)